### ISSUE TITLE
don't have contact event 

### ISSUE THREAD

**Issue by zszen** [jesses/wck#22]
public function UZLPhysic() {
            if (b2world == null) {
                var gravity:V2 = new V2(0,9.8);
                var doSleep:Boolean = true; 
                b2world = new b2World(gravity,doSleep)
                contactListener = new UZLPhysicContactListener
                b2world.SetContactListener(contactListener)
                b2world.SetGravity(new V2(0, 9.8))
                b2world.SetAutoClearForces(true)
                //
            }
        }
the contactListener  don't active when contact

**Comment by jesses** [comment]
Are you setting m_reportBeginContact, m_reportEndContact, etc. on your fixtures?

**Comment by zszen** [comment]
not yet :p let me try again, thx

**Comment by zszen** [comment]
thx, It works well.
